addons/mrp/wizard/mrp_production_split.py

```diff
@@ -46,13 +46,13 @@ def _compute_details(self):
             for _ in range(wizard.counter - 1):
                 commands.append(Command.create({
                     'quantity': quantity,
-                    'user_id': wizard.production_id.user_id,
+                    'user_id': wizard.production_id.user_id or self.env.user.id,
```

Comment:
I think a better fix is to make the `user_id` of `mrp.production.split.line` not required. The user_id of a MO isn't required so it's strange that we require it in MOs created from a split.

Reply:
yah, right